### PR TITLE
Update to Pond v3

### PR DESCRIPTION
**Pull request by wwerner**
Since this has no tests, it's hard for me to verify everything works correctly. Please especially look at 70d7791 in this regard. Version numbers in package.json need to be updated not to use the beta version once Pond v3 is released.

---

This updates to Pond v3.
`ConnectivityStatus` was removed, some changes to event functions, updated docs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,11 +37,11 @@
   },
   "homepage": "https://github.com/actyx-contrib/pond-rx-adapter#readme",
   "peerDependencies": {
-    "@actyx/pond": "^2.5",
-    "rxjs": "> 0"
+    "@actyx/pond": "^3",
+    "rxjs": "> 5"
```
Comment (Alexander89) [inline package.json]: any reason why we do not support 5 anymore? I know one project, who is using 5.5.12

**Review by Alexander89 — COMMENTED** [review]
awesome!!
Nice small impact!!

In terms that the connectivity API is gone, we should go up one major. WHYT?

File: package.json (hunk 1 above)
Comment (wwerner) [reply package.json]: We need `> 5` in `registry`, wouldn't it be good to have the same dependencies?

Comment (Alexander89) [reply package.json]: not really, i think you could use rx-pond even if you are not compatible to registry

**Review by Alexander89 — APPROVED** [review]

**Pull request merged**